src/main/java/org/mafagafogigante/dungeon/schema/rules/BoundIntegerJsonRule.java

```diff
@@ -17,10 +17,10 @@ public void validate(JsonValue value) {
     super.validate(value);
     int intValue = value.asInt();
     if (intValue < minValue) {
-      throw new IllegalArgumentException(value + " is below the allowed minimum " + minValue + ".");
+      throw new IllegalArgumentException(value + " is below the allowed minimum " + minValue);
```

Comment:
It should actually be "the allowed minimum OF ...", my mistake. Let's fix this now.